extension/manifest.json

```diff
@@ -12,15 +12,18 @@
 			"24": "icons/wasp24.png",
 			"32": "icons/wasp32.png"
 		},
-		"default_title": "Apply gradients"
+		"default_title": "Apply gradients",
+		"default_popup": "menu/index.html"
 	},
 	"icons": {
 		"16": "icons/wasp16.png",
 		"48": "icons/wasp48.png",
 		"128": "icons/wasp128.png"
 	},
-	"permissions": ["activeTab", "storage"],
-	"options_ui": {
-                "page":  "options/index.html"
-        }
+	"permissions": [
+		"activeTab",
```

Comment:
Isn't the activeTab permission redundant if we request access to all the webpages via "http[s]://*/"?

Reply:
Maybe? I really don't know. The only reason I needed access to all pages was so that the script would run automatically depending on the user settings. I didn't look into removing any permissions.